tasks/connectors/qualys_was/qualys_was.rb

```diff
@@ -230,7 +235,14 @@ def domain_detail(find_from)
       end
 
       def name(find_from)
-        "#{find_from['qid']}-#{find_from['name']}"
+        if @options[:match_ScannerIdentifier_with_vuln_def_name]
+          return if find_from.nil?
```

Comment:
curious about what would result in find_from being nil

I would say we'd have to return something either way, ingesting nil as the vuln_def name/finding scanner_identifier is likely to cause issues in CVM

Reply:
So in the code, find_from was defined as 

`find_from = data["Finding"]`
and 
data is from 
`finding["data"]`
finding is from findings which a collection from 
`findings_response = qualys_was_get_webapp_findings(token, @options[:qualys_page_size].to_i, page)`

It's true there shouldn't be a situation with nil. But I was thinking more like a safe guard so we don't ingest it? In case the findings_response give us back some nil data? 

Or if it'll make more sense to update the code here 
from: 

`print_debug "there was a problem with the RESPONSE" if findings_response.nil?`

to 

`return if findings_reponses.nil? `

??

Reply:
remove the safeguard nil line as discussed yesterday :)